src/applications/ivc-champva/10-7959C/chapters/applicantInformation.js

```diff
@@ -26,7 +26,10 @@ export const applicantNameDobSchema = {
   uiSchema: {
     ...titleUI('Beneficiary’s name and date of birth'),
     applicantName: fullNameMiddleInitialUI,
-    applicantDOB: dateOfBirthUI({ required: true }),
+    applicantDOB: dateOfBirthUI({
+      required: true,
+      hint: 'For example: January 19 2000',
```

Comment:
We only want this date hint on this screen, so used CSS to hide it everywhere and then just added it back to one location manually.